package com.rodrigodev.xgen.test.organization.packages.packagesForErrorsAreGeneratedFromErrorNames_specialCases.error_name1.error_name2.error___name3_5___;

import com.rodrigodev.xgen.test.organization.packages.packagesForErrorsAreGeneratedFromErrorNames_specialCases.error_name1.error_name2.ErrorName2Exception;
import com.rodrigodev.xgen.test.organization.packages.packagesForErrorsAreGeneratedFromErrorNames_specialCases.RootNameException;

/**
 * Autogenerated by XGen on January 1, 0001.
 */
public class Error___Name3_5___Exception extends ErrorName2Exception {

    public static final ExceptionType TYPE = new ExceptionType();

    private static class ExceptionType extends RootNameException.ExceptionType {

        @Override
        protected RootNameException createException(String message) {
            return new Error___Name3_5___Exception(message);
        }

        @Override
        protected RootNameException createException(String message, Throwable cause) {
            return new Error___Name3_5___Exception(message, cause);
        }
    }

    protected Error___Name3_5___Exception(String message) {
        super(message);
    }

    protected Error___Name3_5___Exception(String message, Throwable cause) {
        super(message, cause);
    }
}
